src/main/steps/common/yourhearings/hearings/template.njk

```diff
@@ -374,6 +374,8 @@
    {% endfor %}
   {% endif %}
   {% if completedHearings.length>=1 %}
+  {# ...
```

Comment:
Please handle all hearing methods in util.ts

Reply:
done